scripts/instances/deploy-dhis2-db-preset.sh

```diff
@@ -12,6 +12,8 @@ NAME=$2
 DATABASE_ID=${DATABASE_ID:-1}
 DATABASE_SIZE=${DATABASE_SIZE:-10Gi}
 INSTANCE_TTL=${INSTANCE_TTL:-""}
+DB_RESOURCES_REQUESTS_CPU=${DB_RESOURCES_REQUESTS_CPU:-250m}
```

Comment:
Are you using all the scripts where you're adding the variables? Unless you're actually using it, I think we're unnecessary cluttering the scripts

Reply:
I think @radnov is following the current pattern of declaring all vars even if they have defaults. If that is what you are talking about. I am not sure why we are doing that. Maybe just to make it easier to discover all parameters 🤷🏻

Reply:
I wasn't aware of that pattern 😁